Plugins/BCClient/Source/BCClientPlugin/Private/BrainCloudRelayComms.cpp

```diff
@@ -378,7 +377,7 @@ void BrainCloudRelayComms::sendRelay(const TArray<uint8> &in_data, const uint64
 
 	// Store inverted player mask
 	uint64 playerMask = 0;
-	for (uint64 i = 0, len = (uint64)MAX_PLAYERS; i < len; ++i)
+	for (uint64 i = 0, length = (uint64)MAX_PLAYERS; i < length; ++i)

```

Comment:
`len` was fine. I always use `i, len`
You can revert this file. I'm about to play in there anyway. You didn't fix anything in it only renamed this :)